test-network-function/generic/suite.go

```diff
@@ -346,6 +351,20 @@ func testGracePeriod(context *interactive.Context, podName, podNamespace string)
 	})
 }
 
+func testNodeSelector(context *interactive.Context, podName, podNamespace string) {
+	ginkgo.It(fmt.Sprintf("Testing pod nodeSelector %s/%s", podNamespace, podName), func() {
+		tester := nodeselector.NewNodeSelector(defaultTimeout, podName, podNamespace)
+		test, err := tnf.NewTest(context.GetExpecter(), tester, []reel.Handler{tester}, context.GetErrorChannel())
+		gomega.Expect(err).To(gomega.BeNil())
+		testResult, err := test.Run()
+
+		gomega.Expect(err).To(gomega.BeNil())
+		if testResult != tnf.SUCCESS {
+			log.Warn(fmt.Sprintf("%s %s has nodeSelector/nodeAffinity, you might want to change it", podName, podNamespace))
```

Comment:
this warning has `podName` and `podNamespace` swapped relative to `ginkgo.It()` message, please format them both the same. The slash-separated format from the `ginkgo.It()` makes the most sense to me.

Reply:
I must not have made myself clear. I meant swap the order of `podName, podNamespace`  to `podNamespace, podName`, rather than rewrite the error to have the pod at the end. Currently the namespace follows the pod name, which does not follow the principle of progressive disclosure.